backend/src/utils/stream.py

```
@@ -302,56 +301,79 @@ async def stream_generator(
+                    return
+                if row["data"]:
+                    yield f"data: {row['data']}\n\n"
+
     except Exception as e:
-        logger.exception(f"Error in stream_from_redis: {e}")
+        logger.exception(f"Error in stream_from_postgres: {e}")
         yield f'data: {{"error": "{str(e)}"}}\n\n'
         yield "data: [DONE]\n\n"
     finally:
-        await redis_client.aclose()
+        await conn.remove_listener(channel_name, lambda *args: None)
```

Ruff (0.14.13)

Unused lambda argument: `args`

(ARG005)